go/vt/vtctl/vtctl.go

```diff
@@ -2330,7 +2330,7 @@ func commandVRWorkflow(ctx context.Context, wr *wrangler.Wrangler, subFlags *fla
 	const defaultMaxReplicationLagAllowed = defaultWaitTime
 
 	cells := subFlags.String("cells", "", "Cell(s) or CellAlias(es) (comma-separated) to replicate from.")
-	tabletTypes := subFlags.String("tablet_types", "primary,replica,rdonly", "Source tablet types to replicate from (e.g. primary, replica, rdonly). Defaults to -vreplication_tablet_type parameter value for the tablet, which has the default value of replica.")
+	tabletTypes := subFlags.String("tablet_types", "in_order:REPLICA,PRIMARY", "Source tablet types to replicate from (e.g. primary, replica, rdonly). Defaults to --vreplication_tablet_type parameter value for the tablet, which has the default value of in_order:REPLICA,PRIMARY.")
```

Comment:
nit: we seem to be using uppercase in the default value, but lowercase in the help text.

Reply:
Where do you see it lower case?

Reply:
ah, you mean more generally than the default value itself e.g. `(e.g. primary, replica, rdonly)`. I prefer capitalized but don't feel too strongly about it.

Reply:
At least started doing that here https://github.com/vitessio/vitess/pull/10040/commits/ebc046911df1391544292fed78a06b569eb603b2 and here https://github.com/vitessio/website/pull/1003/commits/679678ea7cd6cf2ca3fad988c9aac6e03251f7fd